### DISCUSSION TITLE
Custom character is not working

### DISCUSSION THREAD

**Discussion by Gameboss49** [KadeArchive/Kade-Engine#2437] [Q&A]
I'm making a mode for my friend and I tried to add the character to the engine. below is the .xml of the character witch is what I suspect is the culprit.

[New Text Document.txt](https://github.com/KadeDev/Kade-Engine/files/7281302/New.Text.Document.txt)

**Comment by glowsoony** [comment]
me too i try adding vs imposter the green one but its just there floating and wont play animations but i could add it in kade engine 1.7
